### ISSUE TITLE
Accessing globals seems to be broken

### ISSUE THREAD

**Issue by ike709** [wixoaGit/OpenDream#195]
Ex:
```
var/global/screaming = 1

/proc/main()
    if(global.screaming)
        world.log << "AHHHHH"
```

**Comment by wixoaGit** [comment]
There is no `global` keyword currently. `global.screaming` won't work, but `screaming` would.

**Comment by wixoaGit** [comment]
Fixed by 6f4947627c7356b6fdc405146d0e8c41cd8dbf90
